plugin.py

```diff
@@ -79,27 +144,191 @@ class TellRecord(Base):
+                    # This will tie to config, tellMessage. If you want to add more variables, do it here.
+                    _d = {'time_ago': get_timeago(), 'from': t['from'], 'content': t['content']}
+                    if t['private'] is True:
+                        _priv_count += 1
+                        _relay_private = True
+                        _priv_tells.append(self.registryValue("tellMessage").format(**_d))
+                    elif t['private'] is False:
+                        _pub_count += 1
+                        _relay_pub = True
+                        _pub_tells.append(self.registryValue("tellMessage").format(**_d))
+
+                # Relay public tells
+                if _relay_pub:
+                    _m = _public.format(**{
+                        'to': msg.nick,
+                        'pub_count': _pub_count,
+                        'plural': "s" if _pub_count > 1 else ''
+                    })
+                    irc.queueMsg(ircmsgs.notice(_channel, _m))
+
+                    for m in _pub_tells:
+                        irc.queueMsg(ircmsgs.notice(_channel, m))
+
+                # Relay private tells
+                if _relay_private:
+                    _m = _private.format(**{
+                        'to': msg.nick,
+                        'priv_count': _priv_count,
+                        'plural': "s" if _priv_count > 1 else ''
+                    })
+                    irc.queueMsg(ircmsgs.notice(msg.nick, _m))
+
+                    for m in _priv_tells:
+                        irc.queueMsg(ircmsgs.notice(msg.nick, m))
+
+        return msg
+
     def tell(self, irc, msg, args, now, nicks, message):
         """<user1,user2> <message>
     
         Saves a tell for the specified nicks.
         """
-        
-        # TODO:
-        # Channel or private message to bot? Use wrap for the channel? Use ChannelDb?
-        
-        # Insert a Tell in the tell table
-        new_tell = TellRecord(FromNick=msg.nick, ToNick="Ownix", Content=message, Private=0, Read=0, Timestamp=datetime.datetime.now())
-        session.add(new_tell)
+        tell_to = nicks.split(',')
+
+        # Insert tell records per each nick name
+        _dt = datetime.datetime.now()
+        for i in tell_to:
+            new_tell = TellRecord(FromNick=msg.nick, ToNick=i, Content=message, Private=self.pm, Read=0,
+                                  Timestamp=_dt)
+            session.add(new_tell)
+
         session.commit()
-        
-        irc.reply(str("Saving tell '" + message + "' for " + nicks))
+
+        irc.queueMsg(ircmsgs.notice(msg.nick, "Saving tell '" + message + "' for " + nicks))
     tell = wrap(tell, ['now', 'somethingWithoutSpaces', 'text'])
 
+    def skiptells(self, irc, msg, args):
+        """
+        Skip all tells and set to Read.
+        """
+        self.queryTell.flag_all_read(msg.nick)
+
+        irc.queueMsg(ircmsgs.notice(msg.nick, "Beep boop beep. Skipping all tells."))
```

Comment:
`"Marked 4 tells as read"` make this a format string in the config have `flag_all_read` return the num of tells marked.